setup.py

```diff
@@ -39,14 +39,14 @@
     long_description=long_description,
     keywords='dict, dictionary, mapping, bidirectional, bijection, bijective, injective, two-way, 2-way, double, inverse, reverse',
     url='https://github.com/jab/bidict',
-    license='ISCL',
+    license='ISC',
     packages=['bidict'],
     package_data=dict(bidict=['VERSION']),
     zip_safe=True,
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: ISC License (ISCL)',
+        'License :: OSI Approved :: ISC License',
```

Comment:
https://pypi.python.org/pypi?%3Aaction=list_classifiers lists the classifier as I had it, with `(ISCL)` at the end, which is where I got this from. Can you provide a reference explaining why that's wrong in this is right?